pkgs/development/libraries/sundials/3.x.nix

```diff
@@ -12,11 +14,15 @@ stdenv.mkDerivation rec {
 
 
   cmakeFlags = [
-    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples"
+    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples" ] ++
+     stdenv.lib.optional (lapackSupport) [ "-DSUNDIALS_INDEX_TYPE=int32_t"
+      "-DLAPACK_ENABLE=ON"
+      "-DLAPACK_LIBRARIES=[${liblapack}/lib/liblapack${stdenv.hostPlatform.extensions.sharedLibrary},${openblas}/lib/libopenblas${stdenv.hostPlatform.extensions.sharedLibrary},${gfortran.cc.lib}/lib/libgfortran.a]"
   ];
 
-  nativeBuildInputs = [ cmake ];
-  buildInputs = [ python ];
+  nativeBuildInputs = [ cmake ] ++
+                      stdenv.lib.optional (lapackSupport) [ gfortran.cc.lib openblas liblapack ];
```

Comment:
I think `gfortran.cc.lib openblas liblapack` belong in `buildInputs`. We probably also don't need `gfortran.cc.lib` at all, as we pass it explicitly to `LAPACK_LIBRARIES`.